api/catalog_test.go

```diff
@@ -14,19 +15,17 @@ func TestCatalog_Datacenters(t *testing.T) {
 
 	catalog := c.Catalog()
 
-	if err := testutil.WaitForResult(func() (bool, error) {
+	for r := retry.OneSec(); r.NextOr(t.FailNow); {
 		datacenters, err := catalog.Datacenters()
 		if err != nil {
-			return false, err
+			t.Log("catalog.Datacenters: ", err)
```

Comment:
Looks like we have a few different patterns in our tests over the years. Since we have a log that gives the line number, I wonder if it's best to just `t.Log(err)` going forward.